### ISSUE TITLE
Client/BU ID and Matter ID Boxes

### ISSUE THREAD

**Issue by JulieJager** [syncids/syncids-ui#184]
Is there anyway we can make the data in these boxes much larger?  If possible, I would like the checkmarks in the cited column (documents section) to be bold or larger as well.  It is really faint right now.  If this is too difficult then do not worry about it.  Not a priority at this time. :) 

Thanks!!

**Comment by ntgn81** [comment]
@JulieJager , the change is pushed, please give it a try and close this if you're okay with it.

**Comment by JulieJager** [comment]
I see the length of the boxes got bigger but the text did not.  I apologize for not explaining myself better.

I like the boxes bigger, can you make the Client length the same as the Matter length?  And, can you may the text larger or bolded in these boxes?  I hope I explained better.  If this is not possible, its no big deal at this point.

**Comment by ntgn81** [comment]
Sure, I think we can increase the font size, right now it's 12px, maybe 14px or 15px might make it look better without looking out-of-place big

**Comment by ntgn81** [comment]
@yneves , could you give it a try, please?

**Comment by JulieJager** [comment]
I accidently deleted my comment earlier.  Yes, if we can increase the font size to the text that would be great

**Comment by yneves** [comment]
This is fully handled now after #194.
